Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -105,6 +105,7 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
         table.cellLayoutMarginsFollowReadableWidth = false
         table.estimatedRowHeight = SiteTableViewControllerUX.RowHeight
         table.setEditing(false, animated: false)
+        table.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
```suggestion
```

This is not required. Because this is something required of basically everything and we didn't want to type it out every single time on a UI element, we've added the `.build` function to do this automatically. :)